Welolo/client/src/index.js

```diff
@@ -5,32 +5,124 @@ import SuccessfulPayment from './SuccessfulPayment.js';
 
 
 class PaymentForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      fields: {},
+      errors: {},
+    };
+  }
+
+  handleValidation() {
+    let fields = this.state.fields;
+    let errors = {};
+    let formIsValid = true;
+
+    //recipient name
+    if (!fields["recipientname"]) {
+      formIsValid = false;
+      errors["recipientname"] = "Please insert the recipient name";
+    }
+
+    if (typeof fields["recipientname"] !== "undefined") {
+      if (!fields["recipientname"].match(/^[a-zA-Z]+$/)) {
+        formIsValid = false;
+        errors["recipientname"] = "Only letters are allowed";
+      }
+    }
+
+    if (fields["recipientname"].length > 30) {
+      formIsValid = false;
+        errors["recipientname"] = "Sorry, the name is too long";
+    }
+
+    //recipient phone
+    if (!fields["recipientph"]) {
```

Comment:
Xavier was mentioning we had to have it in the +12089997777 format in order for twillio to work. I can add the + in before sending it over to the twilio endpoint, but we'll probably have to allow them to add/change the  1 if we want to support multiple countries. I'm not sure if that's something we want to do, but just something that came to mind :)